content/guides/output-management.md

```diff
@@ -210,8 +210,8 @@ __webpack.config.js__
 +     new CleanWebpackPlugin(['dist']),
       new HtmlWebpackPlugin({
         title: 'Output Management',
-        filename: 'index.html',
-        template: 'src/index.html'
++       filename: 'index.html',
```

Comment:
Just remove those lines, they're not used in the diff before and they're not related to the clean plugin.